README.md

```diff
@@ -9,6 +9,7 @@
 
 ## Prerequisites
 - Create a directory in your repository containing all the markup files along with an `index.rst` file which contains [toctrees](http://www.sphinx-doc.org/en/stable/markup/toctree.html) to link the various documents. You can also provide the index in markdown format although then you won't be able to include other documents such as the `README` into them.
+- If you using the manual index for sub project documentation generation, link to your subprojects using ``` `repository name <./repository name>`_``` in the index.  
```

Comment:
Create a different section for this because links to javadocs, swagger would also go there